OIPA/OIPA/development_settings.py

```diff
@@ -3,6 +3,7 @@
 from OIPA.base_settings import *
 
 DEBUG = True
+FTS = True
```

Comment:
Also add FTS to travis_test_settings.py file to make tests work on Travis build. It fails now there because it uses travis_test_settings.py: 

AttributeError: 'Settings' object has no attribute 'FTS'

Maybe rename to FTS_ENABLED?